content/posts/golang/assignment-to-entry-in-nil-map.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Assignment to Entry in Nil Map"
+summary: "Summary here"
+draft: true
+---
+
+## Overview {#overview}
+
+Example error:
+```
+$ go run main.go
+panic: assignment to entry in nil map
+```
+This panic occurs when you fail to initialize a map properly
+
+## Initial Steps Overview {#initial-steps-overview}
+
+1) [Step 1](#step-1)
+
+2) [Another step](#step-2)
+
+## Detailed Steps {#detailed-steps}
+
+### 1) Use 'make' to initialize the map {#step-1}
```

Comment:
I think these two steps are actually two examples of the same 'solution'. The 'steps' should tell you how to identify where the code you need to fix is. This is a pretty simple runbook though, so you could simplify it to just be a shorter and less structured doc. But I think it's worth keeping consistency even in a simple case.